docs/1-benchpark-setup.rst

```diff
@@ -0,0 +1,27 @@
+=========
+Benchpark setup
+=========
+
+Python 3.6 and git are required to check out and install Benchpark::
+
+  git clone git@github.com:LLNL/benchpark.git $benchpark  
+  benchpark setup [--spack=spack_dir] [--ramble=ramble_dir]
```

Comment:
Benchpark doesn't have these options yet

and this also doesn't describe the required options in the command (which you can get with `benchpark setup --help`).

Reply:
I have talked myself out of these options being a good idea.  Removed.